pyproject.toml

```diff
@@ -1,7 +1,6 @@
 [tool.black]
 line-length = 110
 skip-numeric-underscore-normalization = true
-py36 = true
```

Comment:
--py36 is deprecated and will be removed in a future version. Use --target-version py36 instead.

Since we support py37, I propose to remove it, however the formating is the same with and without the config.